daffodil-tdml/src/main/scala/edu/illinois/ncsa/daffodil/tdml/TDMLRunner.scala

```diff
@@ -553,52 +553,52 @@ abstract class TestCase(testCaseXML: NodeSeq, val parent: DFDLTestSuite)
 
   protected val compiler = Compiler(parent.validateDFDLSchemas)
 
+  private def configFromName(cfgName: String, attrName: String): Option[DefinedConfig] = {
```

Comment:
This change is really a separate feature - I needed this for bitOrder stuff, so I fixed the missing feature of being able to specify config tunables etc. directly in the TDML file, as a default for the file, or per test.